cmd/launcher/launcher.go

```diff
@@ -70,6 +72,8 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 
 	logger := log.With(ctxlog.FromContext(ctx), "caller", log.DefaultCaller, "session_pid", os.Getpid())
 
+	go runOsqueryVersionCheck(ctx, logger, opts.OsquerydPath)
```

Comment:
Do you think we want this in a routine? Probably okay. I guess... having it async is nice, since it won't block and it will gather info.

But if it turns out to matter, having it sync might help. 

Eh, as is!